### ISSUE TITLE
отображение на мобильных устройствах

### ISSUE THREAD

**Issue by mvasin** [ZarNizza/To-Do-App#1]
Салют!

Заметил что на мобилке верстку можно улучшить. На iPhone 8 при открытии URL приложение выглядит вот так:

![IMG_1FC4214BA070-1](https://user-images.githubusercontent.com/12434833/129156525-b34dff08-9ce1-4de9-a1d5-12acc55603ff.jpeg)

**Comment by ZarNizza** [comment]
Yeah, on my POCO (MiBrowser & OperaTouch) also not good... I will fix it asap. Thanks! ))
